core.php

```diff
@@ -247,6 +247,10 @@ function __autoload( $p_class ) {
 	$g_login_anonymous = true;
 }
 
+if( !isset( $g_login_by_bug_token ) ) {
+	$g_login_by_bug_token = false;
+}
+
```

Comment:
Probably not necessary, the variable is initialized when including auth API a couple lines later